docs/.vuepress/components/SliderDoc.vue

```diff
@@ -0,0 +1,46 @@
+<template>
```

Comment:
there should be a live demo part on document. 
It can be implemented in a separate vue file like 
SliderDocLiveDemo.vue

Reply:
Updated live demo in d3c44a61e4d5a2c80e231093710514d2ba83e79a